homeassistant/components/growatt_server/sensor_types/mix.py

```diff
@@ -224,6 +224,7 @@
         api_key="etouser",
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
+        previous_value_drop_threshold=0.2,
```

Comment:
This description doesn't have a state class.

Reply:
Good catch - I meant to add this in as well. I've added in the new commit.